test-common/pom.xml

```diff
@@ -35,6 +35,10 @@
             <artifactId>assertj-core</artifactId>
             <version>3.16.1</version>
         </dependency>
+      <dependency>
+        <groupId>junit</groupId>
```

Comment:
they removed from the transitive dependencies? where was it coming from before?

Reply:
actually from the release note it seems the recommended way is this: https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.4-Release-Notes#junit-5s-vintage-engine-removed-from-spring-boot-starter-test. Can you update accordingly? Thanks!